### DISCUSSION TITLE
Academy - What statements concerning object bricks and classification store are correct?

### DISCUSSION THREAD

**Discussion by lamnc-nfq** [pimcore/pimcore#15994] [Q&A / Support]
Hi everyone,
I spent more time to get the correct answer to this question but still wrong. Please help me correct it
Thank you very much
<img width="545" alt="image" src="https://github.com/pimcore/pimcore/assets/75767767/6c572128-7444-408d-8235-55ca36ec0234">

**Comment by dvesh3** [comment]
@lamnc-nfq Hey! please share your answer with supporting statements so that we can check what's wrong.

**Reply by lamnc-nfq** [reply]
I'm sure for 2 options:
Object bricks support data inheritance.
Classification store keys represent the attributes that can be managed.

How about 2 options below:
Object bricks can be added multiple times to the same data object field ???
Classification store supports all data types of data objects => I'm thinking that it supports some specific types, not at all => Am i correct?

**Reply by patriciomg** [reply]
Certainty leads you to success ;) 

objects bricks cannot be added multiple times , [here](https://pimcore.com/docs/platform/Pimcore/Objects/Object_Classes/Data_Types/Object_Bricks/) look for the paragraph between the first images

about classification stores you'll find your answer [here](https://pimcore.com/docs/platform/Pimcore/Best_Practice/Object_Bricks_vs_Classification_Store/ ), check the table

cheers

**Comment by twin-elements** [comment]
Hi.

11. What statements concerning object bricks and classification store are correct?
 a. Object bricks support data inheritance.
 b. Classification store keys represent the attributes that can be managed.
